bleak/backends/bluezdbus/manager.py

```diff
@@ -166,10 +173,26 @@ def __init__(self):
 
         self._advertisement_callbacks: List[CallbackAndState] = []
         self._device_removed_callbacks: List[DeviceRemovedCallbackAndState] = []
-        self._device_watchers: Set[DeviceWatcher] = set()
-        self._condition_callbacks: Set[Callable] = set()
+        self._device_watchers: Dict[str, list[DeviceWatcher]] = {}
+        self._condition_callbacks: Dict[str, Set[Callable[[Dict[str, Any], None]]]] = {}
```

Comment:
Let's modify and continue to use `DevicePropertiesChangedCallback` here.

Reply:
9c94230